app/api/api/base.rb

```diff
@@ -107,7 +107,8 @@ def render(template_name, **locals)
 
     before do
       key = APIKey.find_by(key: params[:key])
```

Comment:
109 and 110 are both lines intended for redis logging - both could be commented until redis logging can be fixed, as the auth is still done by the `authenticate_app!` call.